### PR TITLE
Example persistent-mysql-haskell usage

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

File: src/App/Db.hs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        { DT.topicPartitionOffsetsTopicPartition  = 1
+        , DT.topicPartitionOffsetsTopic           = "moo"
+        , DT.topicPartitionOffsetsPartitionId     = 2
+        , DT.topicPartitionOffsetsTimestamp       = currentTime
+        , DT.topicPartitionOffsetsOffsetFirst     = 3
+        , DT.topicPartitionOffsetsTimestampFirst  = currentTime
+        , DT.topicPartitionOffsetsOffsetMax       = 4
+        , DT.topicPartitionOffsetsTimestampLast   = currentTime
+        , DT.topicPartitionOffsetsLocationUri     = "url"
+        }
```
Comment (newhoggy) [inline src/App/Db.hs:41]: Insert into table

File: src/App/Db.hs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+      rows <- selectList [DT.TopicPartitionOffsetsPartitionId >. 0, DT.TopicPartitionOffsetsPartitionId <=. 30] []
```
Comment (newhoggy) [inline src/App/Db.hs:43]: Query table

File: src/App/Db.hs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        (dbConfig & _dbConfigHost & T.unpack)
+        (dbConfig & _dbConfigUser & T.encodeUtf8)
+        (dbConfig & _dbConfigPassword & _passwordValue & T.encodeUtf8)
+        (dbConfig & _dbConfigDatabase & T.encodeUtf8)
+
+  _ <- runStderrLoggingT $ withMySQLPool connectionInfo 10 $ \(pool :: Pool SqlBackend) -> liftIO $ do
+    currentTime <- getCurrentTime
+    flip runSqlPersistMPool pool $ do
+      printMigration DT.migrateAll
+      runMigration DT.migrateAll
```
Comment (newhoggy) [inline src/App/Db.hs:30]: Create table

**Comment by AlexeyRaga** [comment]
👍

**Pull request merged**